sdk/documentintelligence/ai-document-intelligence-rest/README.md

```diff
@@ -156,6 +158,50 @@ console.log(result);
 // }
 ```
 
+## Batch analysis
+
+```ts
+import {
+  parseResultIdFromResponse,
+  AnalyzeResultOperationOutput,
+  isUnexpected,
+} from "@azure-rest/ai-document-intelligence";
+
+// 1. Analyze a batch of documents
+const initialResponse = await client
+  .path("/documentModels/{modelId}:analyzeBatch", "prebuilt-layout")
+  .post({
+    contentType: "application/json",
+    body: {
+      azureBlobSource: {
+        containerUrl: batchTrainingFilesContainerUrl(),
+      },
+      resultContainerUrl: batchTrainingFilesResultContainerUrl(),
+      resultPrefix: "result",
+    },
+  });
+console.log("initialResponse: ", initialResponse);
```

Comment:
Is this needed if we already throwing in the error case and printing in the happy case?

Reply:
will remove